specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json

```diff
@@ -529,7 +604,108 @@
               }
             },
             "schema": {
-              "$ref": "#/definitions/StorageError"
+              "$ref": "#/definitions/TableServiceError"
+            }
+          }
+        }
+      },
+      "patch": {
```

Comment:
I have to add this as a "PATCH" operation for now - as autorest does not support "MERGE" operations.
The PATCH operation should still work for a Storage endpoint.